src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -56,13 +62,22 @@ class ContentExplorer extends Component {
          * @param {Array} newFoldersPath
          */
         onEnterFolder: PropTypes.func.isRequired,
+        /** Called when the folders path is updated */
+        onFoldersPath: PropTypes.func,
+        /** Called when the include subfolders toggle is changed */
+        onIncludeSubfoldersToggle: PropTypes.func,
         /**
          * Called when an item is selected
          *
          * @param {Object} selectedItem
          * @param {number} selectedItemIndex
          */
         onSelectItem: PropTypes.func,
+        /** Called when an item is selected
+         *
+         * @param {Object} selectedItem
+         */
+        onSelectedItems: PropTypes.func,
```

Comment:
why do we need this instead of utilizing `onSelectItem` and `onSelectedClick`?

Reply:
There are places where the selectedItems state is altered but these functions are not called. For instance, in the handleSelectAllClick function, the selectedItems in the ContentExplorer state is changed, but no handler is called and I need the parent app to know if those items are selected. So I added the on selectedItems handler that is called in every case that the ContentExplorer's state.selectedItems is changed.
I am not sure why the other handlers you mentioned do not do this, but I was unsure of if I could paste them in these scenarios since I am not entirely sure of where else they are used

Reply:
Both this and the onFoldersPath handlers that I made were both created under the thought process where I knew I needed to know how both of these were changing in the parent component, and I saw some handlers that already existed in ContentExplorer, but they are not being called in every instance where either the selectedItems or foldersPath is being updated. If it can be verified that the other handlers are safe to add to these other situations where the state is being changed, then I can put those in instead

Reply:
gotcha, I think I'm better understanding now